packages/angular/src/generators/application/schema.json

```diff
@@ -99,7 +99,7 @@
     },
     "unitTestRunner": {
       "type": "string",
-      "enum": ["karma", "jest", "none"],
+      "enum": ["jest", "none"],
```

Comment:
The application generator already deletes the test target + karma.conf if the `unitTestRunner` is `jest` or `none`.

The Karma Builder is also an Angular Devkit builder, so it would still work.